File: CasosdePrueba/TestCase4.pas
program Test4;

var
    i, total : INTEGER;

begin
    total := 0;

    { bucle FOR }
    for i := 1 to 5 do
        begin
            total := total + i;
        end;

    { bucle WHILE }
    while total < 20 do
        begin
            total := total + 2;
        end;

    { bucle REPEAT-UNTIL }
    repeat
        total := total - 1;
    until total = 10;

end.

File: CasosdePrueba/TestCase4.c
#include <stdio.h>

int i,total;


int main() {
    total = 0;
    for (i = 1; i <= 5; i++) {
        total = (total + i);
    }
    while (total < 20) {
        total = (total + 2);
    }
    do {
        total = (total - 1);
    } while (!(total == 10));
    return 0;
}
